superset/models/alerts.py

```diff
@@ -100,3 +119,187 @@ class AlertLog(Model):
     @property
     def duration(self) -> int:
         return (self.dttm_end - self.dttm_start).total_seconds()
+
+
+class Observer(Model):
+
+    __tablename__ = "alert_observers"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(150), nullable=False)
+    observer_type = Column(Enum(AlertObserverType))
+    validation_type = Column(Enum(AlertValidationType))
```

Comment:
let's keep it simple and assume that observers collect numbers e.g. floats
add a comment to the observer value